src/register.js

```diff
@@ -0,0 +1,27 @@
+export function findProduct(bikes, code) {
+    for(let i = 0; i < bikes.length; i++) {
+        const bike = bikes[i];
+        if(bike.code === code) {
+            return bike;
+        }
+    }
+    return null;
+}
+
+export function calcLineTotal(quantity, price) {
+    return Number((quantity * price).toFixed(2));
+}
+
+export function calcOrderTotal(cart, bikes) {
+    // declare order total variable
+    let orderTotal = 0;
+    // loop the array of cart (line items)
+    for(let i = 0; i < cart.length; i++) {
+        const cartQty = cart[i];
+        const item = findProduct(bikes, cartQty.code);
```

Comment:
similar comment, but you could have made a const named price intead of item, and avoid dot notating when passing the parm in the calcLineTotal function.